src/web/utils/inputUtils.ts

```diff
@@ -99,8 +101,13 @@ function parseInnerHTMLToText(target: MarkdownTextInputElement): string {
       }
     }
   }
+  text = text.replaceAll('\r\n', '\n');
 
-  return text.replaceAll('\r\n', '\n');
+  // Force letter removal if the input value haven't changed but input type is 'delete'
+  if (text === target.value && inputType.includes('delete')) {
```

Comment:
Coming from https://github.com/Expensify/App/issues/51466, since `inputType` can be undefined it caused an issue here when trying to call `includes` on an `undefined` value.